experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -2539,6 +2539,99 @@ def test_get_changelogs(self):
             ],
         )
 
+    @parameterized.expand(
+        [
+            (NimbusExperiment.Application.FENIX, NimbusExperiment.Application.FENIX, 3),
+            (NimbusExperiment.Application.FENIX, NimbusExperiment.Application.DESKTOP, 0),
+        ]
+    )
+    def test_get_live_multifeature_experiments_for_feature(
```

Comment:
So the method is defined on the model, so I would put the exhaustive tests about all the logical combinations right on the model and then you only need a single integration test on the API.

Reply:
Oh wow I completely misread this, yes this is already exactly correct.  Nevermind me 🤣